channeldb/models/inbound_fee.go

```diff
@@ -0,0 +1,63 @@
+package models
+
+import "github.com/lightningnetwork/lnd/lnwire"
+
+const (
+	feeRateParts = 1e6
+
+	// maxFeeRate is the maximum fee rate that we allow. It is set to 10x
+	// the payment amount.
```

Comment:
Clarified comment. I still wanted to reference the payment amount, because otherwise the comment is not adding any info that isn't in the code already.